spec/unit/action/puppetfile/install_spec.rb

```diff
@@ -2,8 +2,11 @@
 require 'r10k/action/puppetfile/install'
 
 describe R10K::Action::Puppetfile::Install do
-  let(:default_opts) { {root: "/some/nonexistent/path"} }
-  let(:puppetfile) { R10K::Puppetfile.new('/some/nonexistent/path', nil, nil) }
+  let(:default_opts) { { root: "/some/nonexistent/path" } }
+  let(:puppetfile) {
+    R10K::Puppetfile.new('/some/nonexistent/path',
+                         {:moduledir => nil, :puppetfile_path => nil, :force => nil})
```

Comment:
Do you need to specify these? Won't we fall through to the default whether you specify them as `nil` or don't specify them?

Reply:
we don't need to specify these to create a Puppetfile object, but the Install action always passes these options in, so I did so for consistency with how Puppetfile.new is called in the tested class.